app/assets/javascripts/application.js

```diff
@@ -10,7 +10,6 @@
 //= require services/miq_db_backup_service
 //= require directives/scheduler/updateDropdownForFilter
 //= require directives/scheduler/updateDropdownForTimer
-//= require directives/miq_calendar
```

Comment:
@himdel i don't recall why that was removed, maybe just an oversight or result of bad merge/rebase from master